src/components/Table.js

```diff
@@ -74,6 +76,11 @@ export default function Table(props) {
     const selectHacker = (hacker) => {
         props.setSelectedHacker(hacker);
     }
+
+    function totalScore(hackerScore) { // summing up values score
+        const reducer = (accumulator, currentValue) => accumulator + currentValue;
+        return Object.values(hackerScore).reduce(reducer); 
```

Comment:
can this just be put into one line since the function isn't really long

Reply:
i'll cover this in the next commit